dataset/dictionary/uncategorized.json5

```
@@ -1,10 +1,12 @@
 [
   {
-    en: "",
-    ja: "",
-    zhCN: "",
-    pronunciationJa: "",
-    notes: "",
-    notesZh: "",
+    en: "Simulated Universe",
+    ja: "模擬宇宙",
+    zhCN: "模拟宇宙",
+  },
+  {
+    en: "Forgotten Hall",
+    ja: "忘却の庭",
```

I forgot to add `pronunciationJa` here.

```suggestion
    ja: "忘却の庭",
    pronunciationJa: "ぼうきゃくのにわ",
```